glide.lock

```diff
@@ -194,24 +190,32 @@ imports:
 - name: github.com/elazarl/go-bindata-assetfs
   version: 30f82fa23fd844bd5bb1e5f216db87fd77b5eb43
 - name: github.com/emicklei/go-restful
-  version: 892402ba11a2e2fd5e1295dd633481f27365f14d
+  version: 89ef8af493ab468a45a42bb0d89a06fccdd2fb22
   subpackages:
   - log
   - swagger
 - name: github.com/fatih/color
   version: 9131ab34cf20d2f6d83fdc67168a5430d1c7dc23
 - name: github.com/gambol99/go-marathon
-  version: 15ea23e360abb8b25071e677aed344f31838e403
```

Comment:
Keep getting errors after cleaning the Glide cache:

```
[ERROR]	Error scanning k8s.io/client-go/kubernetes/typed/authentication/v1: open /Users/aantonov/.glide/cache/src/https-k8s.io-client-go/kubernetes/typed/authentication/v1: no such file or directory
[ERROR]	This error means the referenced package was not found.
[ERROR]	Missing file or directory errors usually occur when multiple packages
[ERROR]	share a common dependency and the first reference encountered by the scanner
[ERROR]	sets the version to one that does not contain a subpackage needed required
[ERROR]	by another package that uses the shared dependency. Try setting a
[ERROR]	version in your glide.yaml that works for all packages that share this
[ERROR]	dependency.
```
The same error is for `k8s.io/client-go/kubernetes/typed/authorization/v1`, `k8s.io/client-go/kubernetes/typed/autoscaling/v2alpha1`, `k8s.io/client-go/kubernetes/typed/certificates/v1beta1`, `k8s.io/client-go/kubernetes/typed/rbac/v1beta1`, `k8s.io/client-go/kubernetes/typed/settings/v1alpha1`, `k8s.io/client-go/kubernetes/typed/storage/v1` and `k8s.io/client-go/util/flowcontrol` dependencies. :(